lib/3scale_toolbox/entities/base_entity.rb

```diff
@@ -0,0 +1,85 @@
+module ThreeScaleToolbox
+  module Entities
+    module Entity
+
+
+      DEFAULT_PRINTABLE_VARS = %w[
+        id 
+      ].freeze
+      DEFAULT_VERBOSE_PRINTABLE_VARS = %w[
+        id 
+      ].freeze
+      DEFAULT_EXTRA_VARS_NAME = "extra_vars"
+      public_constant :DEFAULT_PRINTABLE_VARS
+      public_constant :DEFAULT_VERBOSE_PRINTABLE_VARS
+      public_constant :DEFAULT_EXTRA_VARS_NAME
+
+      attr_accessor :verbose
+
+      def self.included(base)
+        unless base.const_defined?(:PRINTABLE_VARS)
```

Comment:
I think we can workout this to make simpler.

This is what I would do, but it is open to discussion.
Remove `self.included` method. Entity has `PRINTABLE_VARS` public constant

```
module Entity                            
  PRINTABLE_VARS = %w[                   
        id                               
  ].freeze                               
  public_constant :PRINTABLE_VARS        
                                         
  VERBOSE_PRINTABLE_VARS = %w[           
        id                               
  ].freeze                               
  public_constant :VERBOSE_PRINTABLE_VARS

```

Entities, like Account, may or may not have those configured. But Entities module will have the following method to read printable attributes that will try to read child entity printable attributes and fallback to default defined in Entities module.

```
def printable_attributes                                           
  if verbose                                                       
    self.class.const_get(:VERBOSE_PRINTABLE_VARS, inherit: true)   
  else                                                             
    self.class.const_get(:PRINTABLE_VARS, inherit: true)           
  end                                                              
end                                                                

```